File: week4/day3/labs/helpers/fizzbuzz.js
function fizzBuzz(num1, num2) {
    let grandOutput = [];
    for (let i = 1; i <= 100; i++) {
        let output = "";

        if (i % num1 === 0) {
            output += "fizz";
        }
        if (i % num2 === 0) {
            output += "buzz";
        }

        grandOutput.push(output ? output : i);
    }
    return grandOutput;
}

module.exports = fizzBuzz;

File: week6/day2/fizzbuzz.rb
class FizzBuzz
    attr_writer(:num1, :num2)

    def initialize(num1, num2)
        @num1 = num1
        @num2 = num2        
    end

    def run
        (1..100).map do |i|
            (i % (@num1 * @num2) == 0) ? "fizzbuzz" 
            : (i % @num1 == 0) ? "fizz" 
            : (i % @num2 == 0) ? "buzz" 
            : i 
        end
    end
end


fb = FizzBuzz.new(99, 8)
fb.num1 = 2
puts fb.run